ecl/eclccserver/eclccserver.cpp

```diff
@@ -371,7 +374,7 @@ class EclccCompileThread : implements IPooledThread, implements IErrorReporter,
                 try
                 {
                     StringBuffer loutput;
-                    if (executeLine(lines.item(i+firstCompile), loutput))
+                    if (executeLine(lines.item(i+firstCompile), loutput, (numFailed != 0)))
```

Comment:
Debatable whether it's more helpful to do all the compiles (and thus show all the errors in a single submit rather than having to fix one file's worth at a time?)

Reply:
It is closer to the current behaviour to abort early, and it is definitely not worth linking if the compiles have failed.

Reply:
Agree re linker, don't really care THAT much re compiler... Though it will be random how many actually get compiled.